File: HackerRank/Algorithms/1. Warmup/A_very_big_sum.java
import java.util.Scanner;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here.
           Read input from STDIN.
           Print output to STDOUT.
           Your class should be named Solution. */
        Scanner scan = new Scanner(System.in);
        int size = scan.nextInt();
        int[] arr = new int[size];
        for(int i = 0; i < size; ++i) {
            arr[i] = scan.nextInt();
        }
        scan.close();
        long result = 0;
        for(int i = 0; i < size; ++i) {
            result += arr[i];
        }
        System.out.println(result);
    }
}

File: HackerRank/Algorithms/1. Warmup/A_very_big_sum.cpp
#include <iostream>
using namespace std;

int main() {
    int size;
    ios::sync_with_stdio(false);
    cin >> size;
    int arr[size];
    for(int i = 0; i < size; ++i) {
        cin >> arr[i];
    }
    long long result = 0;
    for(int i = 0; i < size; ++i) {
        result += arr[i];
    }
    cout << result << "\n";
    return 0;
}
